### DISCUSSION TITLE
OS X Yosemite Virtual Machine stuck at login screen spinning circle

### DISCUSSION THREAD

**Discussion by JustSomeGitHubUser693** [utmapp/UTM#6795] [macOS]
I used a vmdk from tech rechard (qemu supports vmdk). ![image](https://github.com/user-attachments/assets/8651d4fe-8a70-4210-80dd-238342ace68f)

**Comment by relikd** [comment]
I had the same issue when installing from the original installer iso. The installation runs through and boots into the newly configured user. But as soon as you restart the machine, it will be stuck at the login screen. I found this workaround: You can enable automatic login for your user. This way, the login screen will not show at all and macOS will login in just fine.
